packages/create-svelte/templates/skeletonlib/.meta.json

```
@@ -1,4 +1,4 @@
 {
-	"title": "Library skeleton project",
+	"title": "Library project",
```

I wonder if we should clarify that it's a library of Svelte components?
```suggestion
	"title": "Svelte library project",
```

---

I feel that's kind of redundant - what else should it be? You're using `create-svelte` to get there.